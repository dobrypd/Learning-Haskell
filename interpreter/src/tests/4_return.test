int a(int a) {
    return 2*a;
}

int main() {
    return a(10);
}
