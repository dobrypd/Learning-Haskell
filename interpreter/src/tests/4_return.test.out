Welcome to Kappa interpreter
20
[(fromList [],fromList [(Ident "a",([Ident "a"],[SStm (JumpS (SjumpFive (Etimes (Econst (Eint 2)) (Evar [Ident "a"]))))])),(Ident "main",([],[SStm (JumpS (SjumpFive (Efunk (Ident "a") [Econst (Eint 10)])))]))],fromList [])]
